LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Configuration/DBPPrivacyConfigurationManager.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  DBPPrivacyConfigurationManager.swift
+//
+//  Copyright © 2023 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import BrowserServicesKit
+import Combine
+import Common
+import PixelKit
+
+public final class DBPPrivacyConfigurationManager: PrivacyConfigurationManaging {
+
+    static let shared = DBPPrivacyConfigurationManager()
```

Comment:
I wonder if we're able to replace the singleton with a single instance created elsewhere, and inject it into any parts of the code that need it, like we do with the dependency providers etc elsewhere.